docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+# Contribute with Docs
+
+All efforts to contribute to the [Docs](https://strapi.io/documentation/3.0.0-beta.x/) are highly appreciated, we recommend [these guidelines](/3.0.0-beta.x/community/contribute-with-docs.html#the-workflow) **and** that you talk to a maintainer before spending much time making a pull request that may not align with the project roadmap.
+
+You are welcome to create tutorials, articles, and videos for your blog or channels without speaking to a maintainer.
+
+## General Guidelines
+
+Please review the following guidelines and recommendations regarding contributing to Docs.
+
+Broadly speaking, if you want to contribute documentation, you will either:
+
+1. Create **new** documentation for a topic not documented, or
+2. Improve existing documentation with additional examples, adding more context, fixing grammar and typos, or doing [translations](/3.0.0-beta.x/community/contribute-with-docs.html#translations).
+
+Each of the two ways above have specific [workflows](/3.0.0-beta.x/community/contribute-with-docs.html#the-workflow) which should be followed to ensure maximum efficiency in your contributions to the documentation.
+
+## Categories of Documentation
+
+There are four different **types of content** existing in the documentation:
+
+a. **Tutorials**
+
+b. **How-tos**
+
+c. **Reference Topics**
+
+d. **Explanations**
+
+Below are explanations for each category of content. The keys are to **know your audience** and **write for them**, both of which are in this contributing guide.
+
+### Tutorials
+
+A **Tutorial** is a complete set of instructions that take the reader through to the completion of a specific end, such as a project or complete working application. The purpose of tutorials is to **teach** general concepts and show how to accomplish specific actions using Strapi.
+
+This _audience member_ can be a developer who is generally speaking is new to JavaScript programming or a developer who is new to programming with Strapi. **Tutorials** are for the newer programmer. More experienced developers, who are new to Strapi, find the **How-tos** better suited to their experience, but they can also follow tutorials.
+
+Examples include the [Building a Static Blog using Gatsby and Strapi](https://blog.strapi.io/building-a-static-website-using-gatsby-and-strapi/) tutorial and the [Cooking a Deliveroo Clone](https://blog.strapi.io/strapi-next-setup/) tutorial.
```

Comment:
Examples include the [Building a Static Blog using Gatsby and Strapi](https://blog.strapi.io/building-a-static-website-using-gatsby-and-strapi/) and the [Cooking a Deliveroo Clone](https://blog.strapi.io/strapi-next-setup/) tutorials.

Reply:
We can do it like this. Done.